### ISSUE TITLE
On fall2024 and fall2024beta, mouseover screen reader descriptions of images seem to be missing

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1843]
I wanted to test the description of the new error message in contracts tabled displays. 
It didn't work, so I checked whether the image descriptions in computing needs all voices were working and they didn't

**Comment by schanzer** [comment]
@ds26gte confirming the descriptions are missing from generated asciidoc. See this screenshot from `histograms`
![image](https://github.com/bootstrapworld/curriculum/assets/77126/0eb3100a-4bdd-4088-b73c-308a015b8763)
